extensions/amp-story/1.0/amp-story.js

```diff
@@ -2582,6 +2581,21 @@ export class AmpStory extends AMP.BaseElement {
     );
   }
 
+  /**
+   * Shows the play/pause icon if there is a playable element in the story
+   * @private
+   */
+  updatePausedIcon_() {
+    const containsPlayableElement = !!this.element.querySelector(
+      'amp-audio, amp-video, [background-audio], amp-story-page.auto-advance-after'
```

Comment:
- `amp-audio` and `amp-video` should ideally use the `page.getAllMedia_()` method, but that sounds very expensive to run as the story is rendering. Maybe a good compromise would be to exclude media in page attachments, by doing `amp-story-grid-layer amp-audio, amp-story-grid-layer amp-video`
- Let's make sure that the `background-audio` is on the right element: `amp-story-page[background-audio]`
- Does that last selector work? It looks like it should be `amp-story-page[auto-advance-after]`. It has the same issues I mentioned in the story-page comment, but worst case scenario the button would be greyed out on all pages so that's fine
- You also need to check for a `background-audio` on the `amp-story` tag: `this.element.hasAttribute('background-audio')`